symfony/plugins/orangehrmPimPlugin/entity/EmployeeSkill.php

```diff
@@ -24,11 +26,12 @@ class EmployeeSkill
     /**
      * @var int
      *
-     * @ORM\Column(name="skill_id", type="integer")
-     * @ORM\Id
-     * @ORM\GeneratedValue(strategy="NONE")
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\Skill", mappedBy="employeeSkills")
```

Comment:
refactor this entity as well. Since this also related to skills.

Reply:
Refactored this relationship